scripts/irods/test/test_resource_types.py

```diff
@@ -831,7 +836,7 @@ def test_attached_mode_explicit_setting_invalid_host__issue_4421(self):
             lib.make_file(file1, 100)
 
             # put small file
-            self.admin.assert_icommand("iput -R %s %s" % (resource_name, file1), 'STDERR', 'USER_SOCK_CONNECT_ERR')  # iput
+            self.admin.assert_icommand("iput -R %s %s" % (resource_name, file1), 'STDERR', 'USER_SOCK_CONNECT_TIMEDOUT')  # iput
```

Comment:
Revert this change.

The reason for this is most likely due to changes to the network. Also, given that there's no iRODS server running at any of the locations defined in the host_list, `USER_SOCK_CONNECT_ERR` is the correct error to expect.

Reply:
Done.